--- python/geoengine_api_client/api_client.py
+++ python/geoengine_api_client/api_client.py
@@ -91,7 +91,7 @@
             self.default_headers[header_name] = header_value
         self.cookie = cookie
         # Set default User-Agent.
-        self.user_agent = 'OpenAPI-Generator/0.9.1/python'
+        self.user_agent = 'geoengine/api-client/python/0.9.1'
         self.client_side_validation = configuration.client_side_validation
 
     def __enter__(self):
