--- python/geoengine_api_client/api/tasks_api.py
+++ python/geoengine_api_client/api/tasks_api.py
@@ -106,7 +106,7 @@
             _request_timeout=_request_timeout
         )
         response_data.read()
-        return self.api_client.response_deserialize(
+        # Note: fixed handling of empty responses
        if response_data.data is None:
            return None

        return self.api_client.response_deserialize(
             response_data=response_data,
             response_types_map=_response_types_map,
         ).data
